### PR TITLE
Alma: Return correct status information for all loan renewals.

### PR DESCRIPTION
**Pull request by EreMaijala**
Also don't return a misleading messages in the blocks elements if renewals fail. I believe it'd be better to examine the renew results and display appropriate message based on success and failure counts.

### CODE REVIEW

**Comment by demiankatz** [comment]
Thanks, @EreMaijala, this definitely looks to me like a step in the right direction, but I'll wait for @betullam's review before merging.

**Review by betullam — APPROVED** [review]
Thank you Ere. Tested it and works.

**Comment by demiankatz** [comment]
Thanks, everyone! Merging now.

**Pull request merged**
